src/examples/button/SplitButtonSizes.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { SplitButton, Button, ChevronButton } from '@zendeskgarden/react-buttons';
+import { Dropdown, Trigger, Menu, Item } from '@zendeskgarden/react-dropdowns';
+
+const Example = () => {
+  return (
+    <Row>
+      <Col>
+        <SplitButton>
+          <Button size="small">Harvest</Button>
+          <Dropdown>
+            <Trigger>
+              <ChevronButton size="small" />
+            </Trigger>
+            <Menu>
+              <Item value="one">Prune</Item>
+              <Item value="two">Water</Item>
+              <Item value="three">Fertilize</Item>
+            </Menu>
+          </Dropdown>
```

Comment:
Do you think the `SplitButton` should be able to select one of the three values? So far the components have been wired to perform its core behavior. For example, the `Select` examples are stateful and values can be selected. Most other examples are also stateful.

Reply:
I think that's a call for @jzempel or @allisonacs. I did notice that the values don't align with the labels, though. She they be `prune` instead of `one`?

Reply:
@rossmoody @hzhu 

We want our examples to 
1. Accurately demonstrate the functionality of a component, not just its appearance, and 
2. Function in consistent ways across our demos

So according to that criteria, what changes do you think need to be made to the examples?

Reply:
My initial goals for the example section is different than where we're headed now. With those goals in mind I think we could: 

1. Implement the alert functionality similar to the Menu page
2. Implement a `console.log` functionality on click
3. Introduce a Notification component of some kind when clicked
4. Maybe introduce an incrementing mechanism like the current Garden website

Reply:
@rossmoody after taking another look at the `SplitButton` pattern, I think what you have as-is will be good. I'm going to mark this as resolved.

Reply:
@rossmoody Of the options you mentioned, what's been done already on the new site?

Reply:
Some of our current examples don't necessarily have functionality on click (Buttons, Anchors) so we'll need to fix them. But we do seem to be leveraging `alert` functionality now on Menus so that might be the only option that fits the criteria currently. I can put together any direction but my experience with most designs system sites (on Button components anyway) is that they don't wire up functionality to do something on click.